### PR TITLE
feat(core): cookie keys configuration

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

add env `OIDC_COOKIE_KEYS` to enable cookie keys customization

## Linear Issue Reference

LOG-2552

## Testing

image

along with localhost sign in

### CODE REVIEW

File: packages/core/src/env-set/oidc.ts [hunk 1]
```diff
@@ -62,11 +64,55 @@ const readPrivateKey = async (): Promise<string> => {
...
+  } catch (error: unknown) {
+    if (noInquiry) {
+      throw error;
+    }
+
+    if (!allYes) {
+      const answer = await inquirer.prompt({
+        type: 'confirm',
+        name: 'confirm',
+        message: `No cookie keys found in env \`${envKey}\`, would you like to generate a new one?`,
```
[the remark this answers is not in the document]
Comment (gao-sun) [reply packages/core/src/env-set/oidc.ts]: updated

[the remark this answers is not in the document]
Comment (gao-sun) [reply packages/core/src/env-set/oidc.ts]: no cookie keys array <-> a new one

[the remark this answers is not in the document]
Comment (demonzoo) [reply packages/core/src/env-set/oidc.ts]: But you didn't mention array, instead you mentioned keys. So `a new one` => a new key 🤔

[the remark this answers is not in the document]
Comment (gao-sun) [reply packages/core/src/env-set/oidc.ts]: yeah so i updated with "cookie keys array", please check again

**Pull request merged**